Adding IPCC AR6 Chapter 3 Fig. 3.9 - Attribution (#2986)
ESMValGroup/ESMValTool: doc/sphinx/source/recipes/recipe_ipccwg1ar6ch3.rst

.. _`Eyring et al., 2021`: https://www.ipcc.ch/report/ar6/wg1/chapter/chapter-3/
.. _`Eyring et al. (2021)`: https://www.ipcc.ch/report/ar6/wg1/chapter/chapter-3/

To reproduce Fig. 3.9 please upload the shapefile of the [AR6 reference regions]
(https://github.com/SantanderMetGroup/ATLAS/tree/v1.6/reference-regions)
([IPCC-WGI-reference-regions-v4_shapefile.zip]
(https://github.com/SantanderMetGroup/ATLAS/blob/v1.6/reference-regions/IPCC-WGI-reference-regions-v4_shapefile.zip))
and store it in `<auxiliary_data_dir>/IPCC-regions/` (the `auxiliary_data_dir`

> is defined in the `config-user.yml` file).

Reviewer: I see both, the text and the linked URL including the parentheses when I preview the .rst file. Maybe double-check if the syntax is correct.

   * e.g. diag_scripts/shared/plot/styles/cmip5.style

#. Script ipcc_ar6/tas_anom_damip.ncl

   *Required settings for script*

   * start_year: start year in figure
   * end_year: end year in figure
   * panels: list of variable blocks for each panel 

   *Optional settings for script*

   * ref_start: start year of reference period for anomalies
   * ref_end: end year of reference period for anomalies
   * ref_mask: if true, model fields will be masked by reference fields
   * plot_units: variable unit for plotting
   * y-min: set min of y-axis
   * y-max: set max of y-axis
   * header: title for each panel
   * title: name of region as part of filename

>    * label: set label as output in an extra file

Reviewer: ```suggestion
   * legend: set labels for optional output of a legend as an extra file
```

   Observational and model datasets were detrended by removing the
   least-squares quadratic trend. 

.. figure::  /recipes/figures/ipccwg1ar6ch3/tas_anom_damip_global_1850-2020.png
   :align:   center

   Figure 3.9: Global, land and ocean annual mean near-surface air

>    temperatures anomalies in CMIP6 models and observations. Timeseries are

Reviewer: ```suggestion
   temperature anomalies in CMIP6 models and observations. Timeseries are
```

   shown for CMIP6 historical anthropogenic and natural (brown) natural-only
   (green), greenhouse gas only (grey) and aerosol only (blue) simulations
   (multi-model means shown as thick lines, and shaded ranges between the 5th
   and 95th percentiles) and for HadCRUT5 (black). All models have been

>    subsampled using the HadCRUT5 observational data mask. Temperatures

Reviewer: ```suggestion
   subsampled using the HadCRUT5 observational data mask. Temperature
```

   anomalies are shown relative to 1950-2010 for Antarctica and relative to

>    1850-1900 for other continents. CMIP6 historical simulations are expand by

Reviewer: ```suggestion
   1850-1900 for other continents. CMIP6 historical simulations are expanded by
```
